zer0-os/zOS
# Add back and forward to address bar

### What does this do?
Adds back and forward buttons to address bar.

### Why are we making this change?
Because we want to go back and forward.

### How do I test this?
Run the tests.
or. Open the app. Click into some feed items. Clicking back should take you back a level. If you go deep, then back, then the forward button should take you deeper.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security? n/a
  1. How will this affect performance? shouldn't.
  1. Does this change any APIs? nope.
